src/components/LandPage.js

```diff
@@ -0,0 +1,75 @@
+import {
+  HeaderStyle,
+  LandPageStyle,
+  SearchBoxStyle,
+} from "../styles/LandPage.styled";
+import { useContext, useEffect } from "react";
+import Context from "./context";
+import MoviesContainer from "./MoviesContainer";
+import NoResult from "./NoResult";
+import { useNavigate } from "react-router-dom";
+
+export const LandPage = () => {
+  let navigate = useNavigate();
+  const a = useContext(Context);
+
+  const API_KEY = "api_key=0decbb49ed9b3d0c6017d7721a14c106";
+  const BASE_URL = "https://api.themoviedb.org/3";
+  const API_URL =
+    BASE_URL + "/discover/movie?sort_by=popularity.desc&" + API_KEY;
+  const searchURL = BASE_URL + "/search/movie?" + API_KEY + "&query=";
+
+  async function getMovie(url) {
+    a.setNoResult(false);
+    a.setShowMovies(false);
+    
+    const res = await fetch(url);
+    const data = await res.json();
+    const result = data.results;
+    if (result.length === 0) {
+      a.setNoResult(true);
+    } else {
+      a.setmovies(result);
```

Comment:
Please use camel casing. It should be setMovies. Check the casing throughout your application and update it accordingly.